### ISSUE TITLE
new users can't create organizations

### ISSUE THREAD

**Issue by jlopp** [grafana/grafana#2184]
I want to set up public access to my Grafana instance so that users can view (but not edit) my organization's dashboards, but also create and share their own custom dashboards. I have the former working, but am hitting a roadblock with the latter. Here's the "users" section of my grafana.ini:
```
[users]
# disable user signup / registration
allow_sign_up = true

# Allow non admin users to create organizations
allow_org_create = true

# Set to true to automatically assign new users to the default organization (id 1)
auto_assign_org = true

# Default role new users will be automatically assigned (if disabled above is set to true)
auto_assign_org_role = Viewer
```
When a newly signed up user tries to create an organization, it just displays an orange error that says "Problem! Access denied"

Am I missing something? How can I allow new users to create their own organizations?

**Comment by torkelo** [comment]
cant replacate this, With same config if I sign up with a new user, then click the organization dropdown and then "New Organization", fill in the name and Org. name field and hit Create button it works. Tried multiple times with different new user sign ups. 

Are you sure you are signed in when you do this?

**Comment by jlopp** [comment]
:-( I'm completely sure that I'm signed in.

If you think it might provide you with any additional insight, you can check my instance at https://statoshi.info/login

I just reproduced it myself and the error I'm seeing in the grafana.log is:
```
2015/06/19 20:15:49 [I] Completed /api/org/ 401 Unauthorized in 15.77284ms
2015/06/19 20:15:49 [I] Completed /api/org/ 401 Unauthorized in 14.641904ms
```

**Comment by jlopp** [comment]
Also, if there is any specific info I can provide (configs, logs, etc) let me know. I already have a blog post written to announce individual user account creation; I'm going to dig into this issue further shortly.

**Comment by jlopp** [comment]
Another odd thing: when I click "Create Org" it sends the create POST, then there's a "login ping" and then it sends a second duplicate "Create Org" POST which also fails with a 401 Unauthorized response.

![screen shot 2015-06-19 at 4 02 05 pm](https://cloud.githubusercontent.com/assets/288011/8261957/b2c7fb04-169c-11e5-8df2-f4c884ead698.png)

**Comment by torkelo** [comment]
Can you show a screenshot of the admin section that shows the backend config? Just to make sure grafana has really been configured to allow org creation. 

I mean the only cause for you issue must be that allowUserOrgCreate

**Comment by torkelo** [comment]
I mean that allow_org_create is false

**Comment by jlopp** [comment]
![screen shot 2015-06-19 at 4 14 31 pm](https://cloud.githubusercontent.com/assets/288011/8262167/4ebfcec8-169e-11e5-8115-1c67802202fe.png)

**Comment by torkelo** [comment]
Found the cause for this, bug in 2.0.2 that requires Org Role to be Admin to create a new Org (fixed in master and 2.1-pre1 builds), 

http://grafana.org/download/builds.html

**Comment by jlopp** [comment]
Sweet!
